miner/worker.go

```diff
@@ -1402,10 +1423,12 @@ func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64)
 	// fetch l1Txs
 	var l1Messages []types.L1MessageTx
 	if w.chainConfig.Scroll.ShouldIncludeL1Messages() {
-		withTimer(l2CommitNewWorkL1CollectTimer, func() {
+		common.WithTimer(l2CommitNewWorkL1CollectTimer, func() {
 			l1Messages = w.collectPendingL1Messages(env.nextL1MsgIndex)
 		})
 	}
+
+	tidyPendingStart := time.Now()
```

Comment:
getPending?

Reply:
Because it collect the pending tx and do some tidy, then put it localTxs. maybe tidy more rational

Reply:
got it.